### ISSUE TITLE
Frontend Templates Not Rendering Properly

### ISSUE THREAD

**Issue by Blastitt** [joostdecock/site#400]
On the pattern documentation pages, some (maybe all) of the directives in the page template are not being rendered and replaced by the desired images, formatting, etc.

### Image Inclusion
![templato-no-rendero](https://user-images.githubusercontent.com/9396048/40519559-1d0dea5e-5f8e-11e8-86be-79f96f030cc4.png)

### Bolding
![boldingrender](https://user-images.githubusercontent.com/9396048/40519629-859db932-5f8e-11e8-848d-85707c62d4e1.png)

### Formatting
![titlerender](https://user-images.githubusercontent.com/9396048/40519630-85ac76c0-5f8e-11e8-95c5-a855570f7c05.png)

No errors in javascript console or connection/downloading issues.

All was working as of May 21, 2018 and stopped working when I refreshed the page on May 24, 2018.

Tested in Chromium and Chrome on Arch Linux.

Can provide further testing if necessary.

**Comment by joostdecock** [comment]
Thank you for the detailed report. We're aware of it, these are the liquid tags that were used by Jekyll. We've migrated our site to Vue.js, and still need to convert our documentation. This is something we'll be working on in the coming days.

Due to the GDPR deadline of May25, we had to publish our new site sooner than we would have liked to, but we'll continue to work on it.

I'm closing this as the matter is already filed under https://github.com/freesewing/site/issues/384
